src/pages/Dashboard/components/ReportTable/TableRow/TableRow.tsx

```diff
@@ -218,16 +223,13 @@ function TableRow({ skill, projections }: TableRowProps): JSX.Element {
                     textAlign="right"
                   >
                     <Box fontSize="10px" fontWeight="600" color="#333333">
-                      {bench.employees &&
-                      bench.employees[index] &&
-                      bench.employees[index].value &&
-                      bench.employees[index].value >= 0.5 ? (
-                        <Text
-                          key={bench.employees[index].name}
-                        >{`${bench.employees[index].name.substring(
-                          0,
-                          bench.employees[index].name.indexOf(" "),
-                        )} ${bench.employees[index].value * 100}%`}</Text>
+                      {employees &&
+                      employees[index] &&
+                      employees[index].value &&
+                      employees[index].value >= 0.5 ? (
```

Comment:
we already filter for value >=0.5 above so we can skip this check.

Reply:
yes good catch!